bin/backward-compatibility-test.js

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+/* eslint-disable no-undef -- avoid errors on process */
+
+const os = require("os");
+const fs = require("fs");
+const path = require("path");
+const { execSync } = require("child_process");
+
+const INSTRUCTIONS =
+  "Usage: node bin/backward-compatibility-test.js <FE_GIT_REF> <BE_GIT_REF> <build|start|test>";
+
+const args = process.argv.slice(2);
+if (args.length !== 3) {
+  console.log(INSTRUCTIONS);
+  process.exit(1);
+}
+
+const FE_GIT_REF = args[0];
+const BE_GIT_REF = args[1];
+const COMMAND = args[2];
+
+const TMP_FOLDER = path.join(process.cwd(), ".tmp");
+const FE_FOLDER = path.join(TMP_FOLDER, "metabase-fe");
+const BE_FOLDER = path.join(TMP_FOLDER, "metabase-be");
+const JAR_PATH = path.join(BE_FOLDER, "target/uberjar/metabase.jar");
+
+const TEST_TO_RUN = [
```

Comment:
nit: as this is an array
```suggestion
const TESTS_TO_RUN = [
```